### DISCUSSION TITLE
Dietpi-Software | Notifiarr

### DISCUSSION THREAD

**Discussion by Maggus101** [MichaIng/DietPi#5296] [Software Request]
#### Formal software information
- Software title | Norifiarr
- Short description | This is the unified client for Notifiarr.com. The client enables content requests from Media Bot in your Discord Server. It also provides reports for Plex usage and system health.
- Official URL (if available) | https://notifiarr.com/
- GitHub URL (if available) | https://github.com/Go-Lift-TV/notifiarr
- (Official) install documentation | https://github.com/Go-Lift-TV/notifiarr/releases

#### Are there similar/alternative software titles available with DietPi-Software?
N/A

#### What makes your requested software better than the above solutions, if available?
N/A

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
1. wget -qO- https://raw.githubusercontent.com/Go-Lift-TV/notifiarr/main/scripts/install.sh | sudo bash
2. nano /etc/notifiarr/notifiarr.conf        
3. sudo systemctl restart notifiarr
4. Somehow fill out the conf file with the software that is installed on the server?
5. i.e. 
[[service]]
  name    = "App: Bazarr"
  type    = "http"
  check   = "http://localhost:6767/series/"
  expect  = "200"
  timeout = "10s"

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
No experience sorry!

**Comment by tjamieg** [comment]
+1 for this, please! I've just installed Radarr & Sonarr through DietPi, but now I would like to install the Notifiarr client along side them.

How does one go about installing "other software" alongside DietPi until is makes the cut to be included by default?

Would the simplest way be to install Docker and use that?

**Comment by MichaIng** [comment]
I moved this to discussions, so you can now vote for it with the up button. Zero votes on FeatHub in the past year, though AFAIK FeatHub was either down or one couldn't login most of the time (like now): https://feathub.com/MichaIng/DietPi/+222
